config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
```

Comment:
I wanted them to be in the env.sample file because I'm much more likely to see that they are values that can be changed in the env file rather than to look through the code for the default values.